src/locales/en.json

```diff
@@ -86,7 +86,7 @@
   "campaign_nft_page_title": "Publish as you write. Collect as you read.",
   "campaign_nft_upcoming": "Next Writing NFT coming on",
   "campaign_nft_view_nft_dashboard": "Visit NFT Dashboard",
-  "cart_purchase_channel": "channel",
+  "cart_purchase_channel": "channel: {from}",

```

Comment:
```suggestion
  "cart_purchase_channel": "Through {from}",

```

Reply:
[Update translation for purchase channel](https://github.com/likecoin/liker-land/pull/1765/commits/333cdcc3e1d68d0c588fb586fbc01d49a56fc577)